HCA-Healthcare/elixir-hl7
# Organize modules sidebar

- resolves #49

## Summary
- groups modules in sidebar
- sets some modules to deprecated
- hides internal modules

## Testing instructions
- Run `mix docs`
- Review sidebar to ensure modules are in the right place on the sidebar, given the specifications in the ticket
- Review modules to ensure all modules in deprecated section have the `moduledoc deprecated: ""` to show the warning at the top.
